tests/test_waypoints_to_commands.py

```diff
@@ -0,0 +1,32 @@
+"""
+Test process
+"""
+import dronekit
```

Comment:
Nit: Add empty line between docstring and import (convention for when there are no system imports).